.travis.yml

```
@@ -1,57 +1,56 @@
 dist: xenial
 language: python
-python: 3.6.8
-
-sudo: false
-
-# The apt packages below are needed for sphinx builds
-addons:
-  apt:
-    packages:
-      - texlive-latex-extra
-      - dvipng
-      - graphviz
-      - libopenblas-base
-      - libopenblas-dev
-
-env:
-  global:
-    - NUMPY_VERSION=1.16
-    - DEV="--upgrade cython git+https://github.com/numpy/numpy git+https://github.com/astropy/astropy"
-
-matrix:
+python: 3.8.5
```

Is pinning bugfix version necessary?
```suggestion
python: 3.8
```

---

I think generally, if you specify only `3.8` it will use `3.8.0`.  At least that has been my experience with Travis.  Also, I recall in the past that it interprets `3.8` as an integer, while it interprets `3.8.5` as a string, and that has different effects.  Maybe if I do `"3.8"` it will get the latest Python 3.8.x?

---

Hmm... `astropy` not using quotes, but then again it also uses `tox`. 🤷 

https://github.com/astropy/astropy/blob/3bae6fa88a9dbc07f1aab696cb47decca1ee21cd/.travis.yml#L76

---

https://travis-ci.com/github/astropy/astropy/jobs/417355775#L237

---

Hmmm... But it is not 3.7.0. 🤷 

You might want to open astropy issue then. One more reason to move away from Travis CI? 😬

---

Maybe there's some default patch that Travis picks if you  just specify the minor release.  Regardless, I don't think it's a problem to specify a later patch.  For `3.8` it installs `3.8.1`:

https://travis-ci.com/github/astropy/astropy/jobs/417355772#L247

Anyway, I'll keep it as-is, as it doesn't hurt.